kfdef/dashboard/overrides/overlays/custom-image/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../base
+
+images:
+  - name: quay.io/opendatahub/odh-dashboard:v0.9
+    newName: quay.io/tcoufal/odh-dashboard
+    newTag: latest
```

Comment:
could we use something that isn't "latest"?

Reply:
I'll tag it the same as the upstream